### ISSUE TITLE
Toolbar bitmaps with mask versus alpha channel

### ISSUE THREAD

**Issue by KeyWorksRW** [KeyWorksRW/wxUiEditor#185]
### Description:

This keeps coming up, so I'm opening a master issue that other issues can refer to.

In Windows, if an app does _not_ have a manifest specifying version 6 of common controls, then you can _not_ use toolbar images with an alpha channel. If you do, the alpha channel is converted to black, so you end up with no transparency. If it does have a mask, then the mask provides the transparency and the toolbar will look fine.

If an XPM file is specified, then it works with or without Visual Styles. If you use Art Provider, it might or might not work as some of the images have an alpha channel whereas other just use a mask. If you use one of our converted PNG header files then the image has an alpha channel.

Note that an image _can_ convert an alpha channel to a mask, but there are several options for how to do this, so **wxWidgets** doesn't do it automatically (see https://docs.wxwidgets.org/3.1/classwx_image.html#a73c3cd71be1f97da115689a0387d197c). In our Convert Image dialog we use `ConvertAlphaToMask(wxIMAGE_ALPHA_THRESHOLD)` which works well for graphics that have a drop shadow, but not so well for images with an alpha channel that contains values > 128.

We have no way of knowing whether the user is using a manifest at all, let alone whether or not it specifies version 6 common controls. As such, it's kind of pointless to try to _fix_ the problem for users that don't use that style setting. If the user wants a drop shadow, the will always get it with an alpha channel on Unix and Max -- so forcing it to a mask just for Windows without the visual style set lowers the image quality on all platforms.

Bottom line: we have to assume the user set Visual Styles if they use a toolbar graphic with an alpha channel. We'll have to deal with the support issue as it comes up (maybe a common QA in discussions?).

**Comment by KeyWorksRW** [comment]
I'm not going to support this. For C++ on Windows, a manifest is needed, but not for Unix/Mac. It also isn't needed for wxPython or wxRuby.
